sdk/storage/Azure.Storage.Blobs.ChangeFeed/api/Azure.Storage.Blobs.ChangeFeed.netstandard2.0.cs

```diff
@@ -33,7 +33,9 @@ public partial class BlobChangeFeedEventData
         internal BlobChangeFeedEventData() { }
         public Azure.Storage.Blobs.ChangeFeed.BlobOperationName BlobOperationName { get { throw null; } }
         public Azure.Storage.Blobs.Models.BlobType BlobType { get { throw null; } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
ChangeFeed is still in preview, can we just change `ClientRequestId` to a string?

Reply:
I think so. Why would we leave empty property behind?